marklogic-client-api-functionaltests/src/test/java/com/marklogic/client/fastfunctest/ClientApiFunctionalTest.java

```diff
@@ -1,1180 +1,1003 @@
-	 * @param uri	Doc Id of the inserted document
-	 * @param content	Doc contents of the inserted document
-	 *
-	 */
-	void SessionChecks(SessionState api_session, String uri, String content);
-
-}
-
-interface TestE2EModuleXQY {
-	/**
-	 * Creates a TestE2EModuleXQY object for executing operations on the database server.
-	 *
-	 * The DatabaseClientFactory class can create the DatabaseClient parameter. A single
-	 * client object can be used for any number of requests and in multiple threads.
-	 *
-	 * @param db	provides a client for communicating with the database server
-	 * @return	an object for executing database operations
-	 */
-	static TestE2EModuleXQY on(DatabaseClient db) {
-		return on(db, null);
-	}
-	/**
-	 * Creates a TestE2EModuleXQY object for executing operations on the database server.
-	 *
-	 * The DatabaseClientFactory class can create the DatabaseClient parameter. A single
-	 * client object can be used for any number of requests and in multiple threads.
-	 *
-	 * The service declaration uses a custom implementation of the same service instead
-	 * of the default implementation of the service by specifying an endpoint directory
-	 * in the modules database with the implementation. A service.json file with the
-	 * declaration can be read with FileHandle or a string serialization of the JSON
-	 * declaration with StringHandle.
-	 *
-	 * @param db	provides a client for communicating with the database server
-	 * @param serviceDeclaration	substitutes a custom implementation of the service
-	 * @return	an object for executing database operations
-	 */
-	static TestE2EModuleXQY on(DatabaseClient db, JSONWriteHandle serviceDeclaration) {
-		final class TestE2EModuleXQYImpl implements TestE2EModuleXQY {
-			private DatabaseClient dbClient;
-			private BaseProxy baseProxy;
-
-			private BaseProxy.DBFunctionRequest req_xqyfunction;
-
-			private TestE2EModuleXQYImpl(DatabaseClient dbClient, JSONWriteHandle servDecl) {
-				this.dbClient  = dbClient;
-				this.baseProxy = new BaseProxy("/ext/TestE2EModuleXQY/", servDecl);
-
-				this.req_xqyfunction = this.baseProxy.request(
-						"TestE2EModuleXQY.xqy", BaseProxy.ParameterValuesKind.SINGLE_ATOMIC);
-			}
-
-			@Override
-			public String xqyfunction(String items) {
-				return xqyfunction(
-						this.req_xqyfunction.on(this.dbClient), items
-				);
-			}
-			private String xqyfunction(BaseProxy.DBFunctionRequest request, String items) {
-				return BaseProxy.StringType.toString(
-						request
-								.withParams(
-										BaseProxy.atomicParam("items", true, BaseProxy.StringType.fromString(items))
-								).responseSingle(false, null)
-				);
-			}
-		}
-
-		return new TestE2EModuleXQYImpl(db, serviceDeclaration);
-	}
-
-	/**
-	 * Invokes the xqyfunction operation on the database server
-	 *
-	 * @param items	provides input
-	 * @return	as output
-	 */
-	String xqyfunction(String items);
-
+package com.marklogic.client.fastfunctest;
```

Comment:
Think I must have hit "format code" on this file, which is why it all appears as a diff. Sigh.